<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmbBoxRenk">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>brown</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblRenk">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekran Rengini Belirleyiniz:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpBoxEG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>121</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Erkek Giyim</string>
   </property>
   <widget class="QRadioButton" name="rdBtnEUstG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Üst Giyim</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdBtnEAltG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt Giyim</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBoxKilo">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnAd">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnSoyad">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAd">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ad :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSoyad">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Soyad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDTarihi">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Doğum Yılı :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblKilo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kilo :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditBilgi">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBilgi">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BİLGİLERİNİZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnListele">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Listele</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Cinsiyet:</string>
   </property>
   <widget class="QRadioButton" name="roBtnK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kadın</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdoBtnE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Erkek</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="grpBoxKUG">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Kadın Üst Giyim</string>
        </property>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Tunik</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Kazak</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpBoxKAG">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Kadın Alt Giyim</string>
        </property>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Etek</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>76</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pantolon</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="grpBoxEUG">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Erkek Üst Giyim</string>
        </property>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>69</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Kazak</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>69</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Gömlek</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpBoxEAG">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Erkek Alt Giyim</string>
        </property>
        <widget class="QCheckBox" name="checkBox_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pantolon</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>şort</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="grpBoxKG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Kadın Giyim</string>
   </property>
   <widget class="QRadioButton" name="rdBtnKUstG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Üst Giyim</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdBtnKAltG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt Giyim</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
